src/main/java/webapp/services/SaleService.java

```diff
@@ -132,4 +132,21 @@ private boolean isValidVAT(int vat) {
 			checkDigitCalc = 0;
 		return checkDigit == checkDigitCalc;
 	}
+	
+	/**
+	 * Implemented so we could remove a sale when a customer is removed from the database
```

Comment:
Don't explain yourself in the code :D Javadoc is to provide info on the method, comments lik this I would put in the report.